client/src/app/map/style.service.ts

```diff
@@ -8,139 +8,90 @@ export class StyleService {
     }
 
     /**
-     * Style function: returns a style based on the layer name
-     * @param layerName
-     * @returns {ol.style.Style}
+     * Get style
+     * @param feature
+     * @returns {any}
      */
     getStyle(feature) {
-        // Default style
-        var style = new ol.style.Style({
-            image: new ol.style.Circle({
-                radius: 4,
-                stroke: new ol.style.Stroke({
-                    color: 'red',
-                    width: 2
-                }),
-                fill: new ol.style.Fill({
-                    color: 'white'
-                })
-            }),
-            stroke: new ol.style.Stroke({
-                color: 'red',
-                width: 1
-            }),
-            fill: new ol.style.Fill({
-                color: 'rgba(255, 255, 255, 0.4)'
-            })
-        });
-        if (feature === 'earthnetworks_lightning_points') {
-            style = new ol.style.Style({
-                text: new ol.style.Text({
-                    text: '\uf0e7',
-                    font: 'normal 20px FontAwesome',
-                    textBaseline: 'Bottom',
-                    fill: new ol.style.Fill({
-                        color: 'yellow',
-                    }),
-                    stroke: new ol.style.Stroke({
-                        color: 'black',
-                        width: 2
+        var style: any;
+        var layerStyle = feature.getProperties().layerStyle;
+        if (!layerStyle){
+            style = getDefaultStyle();
```

Comment:
Does default style depend on the geometry type?

Reply:
This style should work for all geometry types